build.sbt

```diff
@@ -68,6 +68,15 @@ lazy val server = (projectMatrix in file("server"))
       libraryDependencies ++= Dependencies.serverDependencies,
       Compile / packageBin / publishArtifact := false,
       (Compile / compile) := ((Compile / compile) dependsOn printSparkScalaVersion).value,
+      Compile / unmanagedSourceDirectories += {
```

Comment:
Server is not a lib to include but a running app. I would simply build it with Scala 2.13 atm

Reply:
Actually no. Keep it only 2.12. fa-db doesn't support 2.13 yet. :-(

Reply:
You are right - I've changed a thing or two. Now only the agent (and model) is being compiled against all 3 versions, the server is compiled only for 2.12.

The sbt log looks something like this:

```
[info] compiling 7 Scala sources to /Users/ab0222l/Documents/git/atum-service/model/target/jvm-2.12/classes ...
[info] compiling 7 Scala sources to /Users/ab0222l/Documents/git/atum-service/model/target/jvm-2.13/classes ...
[info] compiling 7 Scala sources to /Users/ab0222l/Documents/git/atum-service/model/target/jvm-2.11/classes ...
[info] compiling 25 Scala sources to /Users/ab0222l/Documents/git/atum-service/server/target/jvm-2.12/classes ...
[info] compiling 5 Scala sources to /Users/ab0222l/Documents/git/atum-service/agent/target/spark3-jvm-2.12/classes ...
[info] compiling 5 Scala sources to /Users/ab0222l/Documents/git/atum-service/agent/target/jvm-2.12/classes ...
[info] compiling 5 Scala sources to /Users/ab0222l/Documents/git/atum-service/agent/target/jvm-2.13/classes ...
[info] compiling 5 Scala sources to /Users/ab0222l/Documents/git/atum-service/agent/target/spark2-jvm-2.12/classes ...
[info] compiling 5 Scala sources to /Users/ab0222l/Documents/git/atum-service/agent/target/spark3-jvm-2.13/classes ...
[info] compiling 5 Scala sources to /Users/ab0222l/Documents/git/atum-service/agent/target/jvm-2.11/classes ...
[info] compiling 5 Scala sources to /Users/ab0222l/Documents/git/atum-service/agent/target/spark2-jvm-2.11/classes ...
```